streaming/src/main/scala/io/gearpump/streaming/executor/Executor.scala

```diff
@@ -179,11 +187,28 @@ class Executor(executorContext: ExecutorContext, userConf : UserConfig, launcher
 
     case query: QueryExecutorConfig =>
       sender ! ExecutorConfig(systemConfig)
+
+    case registered: TaskRegistered =>
+      LOG.debug(s"Register task ${registered.taskId} succeed")
```

Comment:
Is this log important? Do we need to make it a INFO level?

What if some TaskRegistered  message is lost ,will there be some problem?